### PR TITLE
Logging notes for 2.4.1

### PR DESCRIPTION
**Pull request by amread**

### CODE REVIEW

File: _docs/upgrading/upgrading-to-2-4.md [hunk 1]
```diff
@@ -271,3 +269,121 @@ rstudio:
...
+
+The default S3 long term storage may be overridden with the following manifest configuration:
+
+```yaml
+fluentd:
+  file:
+    enabled: true
+  s3:
+    enabled: false
+    mount: /domino/logjam/data # optional override of default
```
Comment (fernandoacorreia) [inline _docs/upgrading/upgrading-to-2-4.md]: Is the mount for `file` defined under the `s3` key or is that line in the wrong group?

File: _docs/upgrading/upgrading-to-2-4.md [hunk 2]
```diff
@@ -271,3 +269,121 @@ rstudio:
...
+```
+
+The file sink requires the volume to be mounted at the same path on
+all frontend machines and must be writable by the `fluent` user (or
+the user the fluentd process runs as):
+
+```sh
+chown fluent:fluent /domino/logjam/data
+```
```
Comment (fernandoacorreia) [inline _docs/upgrading/upgrading-to-2-4.md]: Is that a manual step that must be done after deployment? Doesn't the salt code do that?

File: _docs/upgrading/upgrading-to-2-4.md [hunk 3]
```diff
@@ -271,3 +269,121 @@ rstudio:
...
+com.cerebro.domino.logjam.file.enabled: {true|false}
+```
+
+Remember to restart frontends for CentralConfig settings to take
+effect.
+
+The logging system uses the Docker label and Kubernetes annotation
+`logKey` to dispatch logs to the appropriate bucket. Runs started
+before the upgrade will not have log output collected for any time
+they ran after the upgrade. This includes APIEv1, interactive sessions
```
Comment (fernandoacorreia) [inline _docs/upgrading/upgrading-to-2-4.md]: "interactive sessions" are now called "workspaces"

File: _docs/upgrading/upgrading-to-2-4.md (hunk 2 above)
Comment (amread) [reply _docs/upgrading/upgrading-to-2-4.md]: I didn't set up any salt code to deal with this mount point. Administrators will need to work out how to get a volume mounted appropriately, it is important that they understand the volume must be writable by the fluent user. I'll write something to that effect.

Comment (amread) [reply _docs/upgrading/upgrading-to-2-4.md]: I removed the chown example, I think it distracts from the point I'm trying to make, which is that admins must mount a volume that is writable to the fluentd process.

**Comment by amread** [comment]
@chrismyang @fernandoacorreia should we merge this?

**Pull request merged**